### PR TITLE
feature/getRoomTerrain

### PR DESCRIPTION
**Pull request by sanados**
### Brief Description

adding getRoomTerrain function
adding RoomTerrain as return of getRoomTerrain (only typed get method)

### Checklists

- [X] Test passed
- [X] Coding style (indentation, etc)
- [X] Edits have been made to `src/` files not `index.d.ts`
- [X] Run `npm run dtslint` to update `index.d.ts`

### CODE REVIEW

File: dist/index.d.ts [hunk 1]
```diff
@@ -1962,14 +1962,18 @@ interface GameMap {
      */
     getTerrainAt(x: number, y: number, roomName: string): Terrain;
     /**
-     * Get terrain type at the specified room position. This method works for any room in the world even if you have no access to it.
-     * @param pos The position object.
+     * Get room terrain for the specified room. This method works for any room in the world even if you have no access to it.
+     * @param roomName String name of the room.
```
Comment (resir014) [inline dist/index.d.ts]: I think you might have added this docblock for the wrong method?

Comment (resir014) [reply dist/index.d.ts]: Disregard this, meant to add a note at the other file.

File: src/map.ts [hunk 2]
```diff
@@ -54,14 +54,18 @@ interface GameMap {
      */
     getTerrainAt(x: number, y: number, roomName: string): Terrain;
     /**
-     * Get terrain type at the specified room position. This method works for any room in the world even if you have no access to it.
-     * @param pos The position object.
+     * Get room terrain for the specified room. This method works for any room in the world even if you have no access to it.
+     * @param roomName String name of the room.
      */
     getTerrainAt(pos: RoomPosition): Terrain;
```
Comment (resir014) [inline src/map.ts]: I think you might have added this docblock for the wrong method?

**Comment by sanados** [comment]
No idea how I screwed those up.
Fixed.

**Pull request merged**